wix-media-v2/wix-media-v2/Files/guides/files-intro.md

```diff
@@ -60,3 +57,33 @@ Elevating a function allows it to be called by any site visitor.
 Exercise caution to prevent security vulnerabilities.
 </p>
 </blockquote>
+
+## Terminology
+
+* **Generate File Download Url** vs. **Generate Files Download Url**
```

Comment:
This isn't terminology. Terminology is term-definition pairs. This is reference information.

I don't have an opinion yet on where it should go; let's sync on this one after the review.